docs/fedora-releases-guide/what-to-know.md

```diff
@@ -0,0 +1,106 @@
+
+### Manual CLI Operations
+
+During service outages, maintain workflow using Packit CLI:
+
+1. Install [Packit CLI](https://packit.dev/docs/cli)
+2. [Configure required tokens](https://packit.dev/docs/configuration#user-configuration-file)
+3. Verify setup (if not already working in service):
+   - Follow [dist-git onboarding guide](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding)
+   - Validate config with `packit validate-config`
+
+#### Troubleshooting Common Issues
+
+**Missing pull request for release:**
+
+Use [`packit pull_from_upstream`](https://packit.dev/docs/cli/pull-from-upstream) when [release-monitoring](https://release-monitoring.org/) detects updates but pull requests aren't created.
+
+**Missing Koji build:**
+
+The release pull request is merged (and contains a change in the version value) but a Koji build isn't triggered.
+
+- Check [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters)
+- Trigger build manually with [`packit build in-koji`](https://packit.dev/docs/cli/build/in-koji)
+
+When using Packit CLI to create pull requests, ensure your FAS account is added to the `allowed_pr_authors` list to trigger Koji builds automatically. This enables the expected build workflow for maintainer-created pull requests.
+
+**Missing Bodhi update:**
+
+The Koji build is created but there is no Bodhi update.
+
+- Packit does not create Bodhi updates for `rawhide`, they are automatically created by Bodhi.
+- Verify [`allowed_builders`](https://packit.dev/docs/configuration/downstream/bodhi_update#optional-parameters)
+- Create manually using [`packit create-update`](https://packit.dev/docs/cli/create-update)
+
+For updates on non-rawhide branches, verify your FAS account is included in the `allowed_builders` list. When submitting Koji builds via Packit CLI, automatic Bodhi update creation requires your FAS account to be explicitly authorized in this configuration.
+
+## Customization options
+
+Packit works best with simple package structures:
+- Single source
+- Single patch
+- Straightforward spec file
+
+For complex packages, customize Packit using:
+- [Actions/hooks](https://packit.dev/docs/configuration/actions)
+- Reference [configuration examples](https://packit.dev/docs/configuration/examples#examples-for-actions)
+
+### `changelogs`
+Customize `changelogs` generation [following these examples](https://packit.dev/docs/configuration/examples#custom-changelog-generation)
+
+### side tags - releasing multiple packages in a single update
+[Follow this guide](https://packit.dev/docs/fedora-releases-guide/releasing-multiple-packages) to enable builds in `side-tags`.
+
+## Ecosystem Awareness
+
+Before adoption, review similar packages in the same ecosystem for best practices and configuration patterns.
```

Comment:
Is it worth making a list of ecosystem and point to an example package for every one of them?
I am not sure I could list many ecosystems. `rust` for sure but I would not know which package to check. And then, python? java? gnome? I don't know.

Reply:
that could be nice, or also linking sourcegraph.com that users can use for easy searching?